contracts/modules/common/Utils.sol

```diff
@@ -31,6 +31,7 @@ library Utils {
     bytes4 private constant ERC721_SAFE_TRANSFER_FROM_BYTES = bytes4(keccak256("safeTransferFrom(address,address,uint256,bytes)"));
     bytes4 private constant ERC1155_SAFE_TRANSFER_FROM = bytes4(keccak256("safeTransferFrom(address,address,uint256,uint256,bytes)"));
 
+    bytes4 private constant OWNER_SIG = 0x8da5cb5b;
```

Comment:
Maybe use `bytes4 private constant OWNER_SIG = bytes4(keccak256("owner()"));` to be consistent with our other constants?

Reply:
This raises a `TypeError: Only direct number constants and references to such constants are supported by inline assembly.` error

Reply:
We could probably add `bytes4 sig = OWNER_SIG;` and use `sig` in the assembly to fix the error but no need to add the extra gas.

If we do one more iteration on this PR then we could add `bytes4(keccak256("owner()"));` above the `OWNEER_SIG` declaration to make the code more readable.